package.json

```diff
@@ -207,6 +207,39 @@
             "references": true
           }
         },
+        "rubyLsp.featuresConfiguration": {
+          "description": "Turn on/off specific features from request",
+          "type": "object",
+          "properties": {
+            "codeLens": {
+              "description": "Customize code lens features",
+              "type": "object",
+              "properties": {
+                "enableAll": {
+                  "type": "boolean"
+                },
+                "gemfileLinks": {
+                  "type": "boolean"
```

Comment:
Let's add a description here since people may not know immediately what `gemfileLinks` refers to.